core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -147,6 +148,9 @@ protected void onJoinComplete(
   ) {
     assignmentSnapshot = SchemaRegistryProtocol.deserializeAssignment(memberAssignment);
     listener.onAssigned(assignmentSnapshot, generation);
+    if (nodeCountMetric != null) {
+      nodeCountMetric.set(assignmentSnapshot.nodes().size());
```

Comment:
Maybe add a null check for `nodes()` as well?  In case of upgrades from an older version this may be the case.

Reply:
Fixed, thank you !